specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/networkManager.json

```diff
@@ -0,0 +1,710 @@
+            "$ref": "#/parameters/ListSkipTokenParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/EffectiveVirtualNetworksParameter"
+            },
+            "description": "Effective Virtual Networks Parameter."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns list of effective Virtual Networks.",
+            "schema": {
+              "$ref": "./networkManagerGroup.json#/definitions/EffectiveVirtualNetworksListResult"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Network Manager": {
+            "$ref": "./examples/NetworkManagerEffectiveVirtualNetworksListByNetworkManager.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkManager": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkManagerProperties",
+          "description": "The network manager properties"
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "The Managed Network resource"
+    },
+    "NetworkManagerListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkManager"
+          },
+          "description": "Gets a page of NetworkManager"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets the URL to get the next set of results."
+        }
+      },
+      "description": "Result of the request to list NetworkManager. It contains a list of groups and a URL link to get the next set of results."
+    },
+    "NetworkManagerProperties": {
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "A friendly name for the network manager."
+        },
+        "description": {
+          "type": "string",
+          "description": "A description of the network manager."
+        },
+        "networkManagerScopes": {
+          "properties": {
+            "managementGroups": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              },
+              "description": "List of management groups."
+            },
+            "subscriptions": {
```

Comment:
What could be the max size of subscriptions and managementGroups array? Can all the list fits in a single request?

Reply:
So far we don't have size limitation for subscriptions  and managementgroups. And it is allowed to put both lists in single request